lang/csharp/versions.props

```diff
@@ -20,41 +20,44 @@
 -->
 
```

Comment:
The tabulation in some places  seems to be incorrect.

Reply:
Made it more consistent.  It's because we used 4 space tabs and not 2 space tabs.  It's defaulting from the editorconfig.

Reply:
The issue was not the spaces vs tabs, but some lines were more tabbed than the others. The xml comment first line seems to be overtabbed IMO

Reply:
Actually the whole XML seems weirdly tabbed

Reply:
Did you view the file here? https://github.com/apache/avro/blob/5e318d76cf68b5661f817c43d7158545ba8fbd89/lang/csharp/versions.props

I think it's just the way the compare is showing it.

Reply:
Default in VS is tabs at a spacing of 4.

Reply:
Updated so that is is now 2 spaces.  This has to be manually changed in Visual Studio.  Apparently XML is not defined in our editorconfig.

Would need to add the following:
```
# XML files
[*.xml]
indent_style = space
indent_size = 2
trim_trailing_whitespace=true
```